### PR TITLE
Updated LICENSE from (C) to ©

### PR DESCRIPTION
**Pull request by Sumit1777**
Updated LICENSE from (C) to ©

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2015-2018 Kong Inc. (https://konghq.com)
+Copyright © 2015-2018 Kong Inc. (https://konghq.com)
```
Comment (tlblessing) [inline LICENSE:3]: Should this be updated to year 2020? @hbagdi

Comment (hbagdi) [reply LICENSE:3]: Indeed. And `(c)` is good as is.

Comment (tlblessing) [reply LICENSE]: Thank you @hbagdi !

**Review by tlblessing — APPROVED** [review]
We are updating year only per Harry and not using the symbol.

**Comment by tlblessing** [comment]
Thank you @Sumit1777 the PR and bringing this file to our attention.

**Pull request merged**